[2024-06-10T01:30:49.282+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_toll_data_17.load_to_database scheduled__2024-06-08T00:00:00+00:00 [queued]>
[2024-06-10T01:30:49.348+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_toll_data_17.load_to_database scheduled__2024-06-08T00:00:00+00:00 [queued]>
[2024-06-10T01:30:49.352+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-06-10T01:30:49.412+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): load_to_database> on 2024-06-08 00:00:00+00:00
[2024-06-10T01:30:49.426+0000] {standard_task_runner.py:57} INFO - Started process 100 to run task
[2024-06-10T01:30:49.471+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_toll_data_17', 'load_to_database', 'scheduled__2024-06-08T00:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/ETL_toll_data.py', '--cfg-path', '/tmp/tmpgkjgbjdp']
[2024-06-10T01:30:49.494+0000] {standard_task_runner.py:85} INFO - Job 219: Subtask load_to_database
[2024-06-10T01:30:49.785+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_toll_data_17.load_to_database scheduled__2024-06-08T00:00:00+00:00 [running]> on host bec75c81cc74
[2024-06-10T01:30:50.737+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dummy@gmail.com' AIRFLOW_CTX_DAG_OWNER='naveed' AIRFLOW_CTX_DAG_ID='ETL_toll_data_17' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-06-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-08T00:00:00+00:00'
[2024-06-10T01:30:50.766+0000] {logging_mixin.py:149} INFO - database connected successfully
[2024-06-10T01:30:57.018+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-06-10T01:30:57.042+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=ETL_toll_data_17, task_id=load_to_database, execution_date=20240608T000000, start_date=20240610T013049, end_date=20240610T013057
[2024-06-10T01:30:57.111+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-06-10T01:30:57.187+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
